frontend/src/metabase/static-viz/components/XYChart/shapes/AreaSeries.tsx

```diff
@@ -28,19 +30,20 @@ export const AreaSeries = ({
   if (areStacked) {
     return (
       <AreaSeriesStacked
-        series={series}
+        series={multipleSeries}
         // Stacked charts work only for a single dataset with one dimension and left Y-axis
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         yScale={yScaleLeft!}
-        xAccessor={xAccessor}
+        xAccessor={xAccessor as unknown as StackedDatumAccessor}
```

Comment:
we're feeding a datum accessor that expects the argument to be a `SeriesDatum`, an array of length 2, but AreaSeriesStacked requires the accessor that supports the argument to be `StackedDatum`, an array of length 3.

It appears that this is working fine, as we always pass a `lineAccessor` to `<AreaSeries />`
https://github.com/metabase/metabase/blob/7406d6fca01deedd71d04641b737a9861c0a03f3/frontend/src/metabase/static-viz/components/XYChart/utils/scales.ts#L51-L52

and it's define that it only expect a `SeriesDatum`, an array of length 2, but we never use the 3rd item in the array, so they're actually compatible.